git-gui: add page (#12258)
tldr-pages/tldr: pages/common/git-gui.md

# git gui

> > A GUI for making new commits, amending existing ones, creating branches, performing local merges, and fetching/pushing to remote repositories.

Reviewer: Maybe we could shorten or break the description a bit.

```suggestion
> A GUI for making new and amending commits, creating branches, performing local merges, and fetching/pushing to remote repositories.
```
(or)

```suggestion
> A GUI for Git to manage branches, commits, and remotes.
```

Feel free to reword it differently too.

> See also: `git-cola`, `gitk`.
> More information: <https://git-scm.com/docs/git-gui>.

- Launch the GUI:

`git gui`

- Show a specific file with author name and commit hash on each line:

`git gui blame {{path/to/file}}`

- Open `git gui blame` in a specific revision:

`git gui blame {{revision}} {{path/to/file}}`

> - Open `git gui blame` by scrolling the view to center on a specific line:

Reviewer: Agreed, this looks much better.

> `git gui blame {{revision}} Makefile`
> 
> - Open `git gui blame` scrolling the view to center on a specific line:
> 
> `git gui blame --line={{line}} Makefile`

Reviewer: Maybe we can make it a bit more generalized.
```suggestion
`git gui blame {{revision}} {{filename}}`

- Open `git gui blame` by scrolling the view to center on a specific line:

`git gui blame --line={{line}} {{filename}}`
```

- Open a window to make one commit and return to the shell when it is complete:

`git gui citool`

> - Open `git gui citool` in the Amend Last Commit mode:

Reviewer: ```suggestion
- Open `git gui citool` in the "Amend Last Commit" mode:
```
